Description: Change helper.rb
 Comment the line starting with Bundler.
Author: Sagar Ippalpalli <i.vikram15@gmail.com>
#Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Forwarded: no
#Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
#Reviewed-by: <name and email of a reviewer, optional>
#Last-Update: <YYYY-MM-DD, last update of the meta-information, optional>
---
#This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/helper.rb
+++ b/tests/helper.rb
@@ -8,6 +8,6 @@
 require 'fog/test_helpers'
 require File.expand_path('../../lib/fog/dynect', __FILE__)
 
-Bundler.require(:test)
+#Bundler.require(:test)
 
 Excon.defaults.merge!(:debug_request => true, :debug_response => true)
